src/components/CommentList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getCommentsById } from "./utils/api";
+import CommentCard from "./CommentCard";
+
+const CommentList = props => {
+  const { article_id } = props;
+  const [comments, setComments] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
```

Comment:
Either this should default to true, or needs to be set true before fetching the comments, because currently is isLoading is always false and the message never shows up.